scripts/sim-docker/entrypoint.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 export jaia_fleet_index=$JAIA_SIM_FLEET
 export JAIA_REST_API_PRIVATE_KEY=simulation
-cd /jaiabot/build/amd64/share/jaiabot/web/server/
-source /jaiabot/build/web_dev/python/venv/bin/activate
+cd /usr/share/jaiabot/web/server
+source /usr/share/jaiabot/python/venv/bin/activate
 ./app.py &
 cd /jaiabot/build/amd64/share/jaiabot/web/rest_api/
-./app.py -e 1:localhost:40000 &
-cd /jaiabot/config/launch/simulation/
+./app.py localhost &
```

Comment:
Should read something like:

```
cd /usr/share/jaiabot/web/rest_api
export JAIA_REST_API_PRIVATE_KEY=""
./app.py &
```

Reply:
Specifically the path is wrong (`/usr/share/jaiabot/web/rest_api` not `/jaiabot/build/amd64/share/jaiabot/web/rest_api/`) and there's no `localhost` argument

Reply:
To test just open http://localhost:9092/jaia/v1/status/all - you'll get all the bot/hub statuses in JSON